pkgs/development/tools/fdroidserver/default.nix

```diff
@@ -82,13 +83,8 @@ buildPythonApplication rec {
     pyyaml
     qrcode
     requests
-    (ruamel-yaml.overrideAttrs (old: {
-      src = fetchPypi {
-        pname = "ruamel.yaml";
-        version = "0.17.21";
-        hash = "sha256-i3zml6LyEnUqNcGsQURx3BbEJMlXO+SSa1b/P10jt68=";
-      };
-    }))
+    ruamel-yaml
```

Comment:
Why do you remove this override? Given that F-Droid buildserver is still running on Debian bookworm which has 0.17.21 we still need to use this version.

Reply:
because I couldn't find a reason to fix the version anywhere.

Reply:
See https://github.com/NixOS/nixpkgs/pull/254518.